### ISSUE TITLE
Virgin mode megane iv rlink 2

### ISSUE THREAD

**Issue by maks88888** [cedricp/ddt4all#303]
Megane iv 2016
Virgin mode rlink 2 
hello
I selected the virgin mode (developer mode).
Please help to exit this virgin mode.
Thanks

**Comment by maks88888** [comment]
display 7 '
thank you very much for help

**Comment by shrlnm** [comment]
> Please help to exit this virgin mode.

e-mail?

**Comment by maks88888** [comment]
Maks888@o2.pl

**Comment by BillyJac** [comment]
Hi @shrlnm ,
Firstly, thank you so much for the amazing work with all your help (Read ALOT of your comments.lol)
I have Megane IV 2017, 
Pressed HIGH in Developer mode, then attempted manual firmware update 2.300 -> 3.3 
Unit started update and restarted - it never came on again. Screen dead
I have ELM327 (Wifi), Win10

Again, thank you for your help
billy.jacobie@gmial.com

**Comment by engindalmis** [comment]
engindalmis26@hotmail.com

**Comment by shrlnm** [comment]
> [engindalmis26@hotmail.com](mailto:engindalmis26@hotmail.com)

Could you please explain your issue

**Comment by engindalmis** [comment]
I accidentally pressed the mid key on the developer side.  my car is a 2016 model 1.5 dc hatcback.the size of my screen is 8.7 inc. the screen turned 90 degrees.and also white spots appeared on the screen. thanks for your help

> shrlnm <notifications@github.com> şunları yazdı (22 Ara 2019 10:00):
> 
> Could you please explain your issue

**Comment by shrlnm** [comment]
@engindalmis 
Ok
First try this
https://gitlab.com/py_ren/pyren/-/wikis/how%20to%20rlink2%20black%20screen

**Comment by engindalmis** [comment]
the screen is straightened, but there are lines extending downwards on the left side. how can we fix this?

iPhone’umdan gönderildi

shrlnm <notifications@github.com> şunları yazdı (22 Ara 2019 12:12):

﻿

@engindalmis<https://github.com/engindalmis>
Ok
First try this
https://gitlab.com/py_ren/pyren/-/wikis/how%20to%20rlink2%20black%20screen

**Comment by engindalmis** [comment]
iPhone’umdan gönderildi

Engin Dalmış <engindalmis26@hotmail.com> şunları yazdı (22 Ara 2019 12:52):

﻿ the screen is straightened, but there are lines extending downwards on the left side. how can we fix this?

iPhone’umdan gönderildi

shrlnm <notifications@github.com> şunları yazdı (22 Ara 2019 12:12):

﻿[cid:FBAAD7F8-1520-41AC-BD12-3D1A781A776D-L0-001]

@engindalmis<https://github.com/engindalmis>
Ok
First try this
https://gitlab.com/py_ren/pyren/-/wikis/how%20to%20rlink2%20black%20screen

**Comment by ouguena** [comment]
![버진모드2](https://user-images.githubusercontent.com/62410347/77132824-24a20480-6aa4-11ea-90ba-631a05d9cfb3.png)

**Comment by tyrogb** [comment]
Hello guys, can someone help me? I dont know exactly what I pressed maybe virgin mode, maybe the high mode. I tried with ddt4all that was slow for me for some reason cannot do anything with it, and then a tried pyron that didnt work also. 
![IMG_20220718_230634](https://user-images.githubusercontent.com/110061792/181100399-7e5887e8-3307-47c2-a12f-b646e8de889e.jpg)

**Comment by AlMeu** [comment]
As it see you can not connect at all to your unit.
Do you have still sound in speakers?
The USB ports are still working?

**Comment by tyrogb** [comment]
Yes everithing is working except the display.

**Comment by AlMeu** [comment]
> First try this
> https://gitlab.com/py_ren/pyren/-/wikis/how%20to%20rlink2%20black%20screen
Try this

**Comment by tyrogb** [comment]
I tried these none of the 4 works for me

**Comment by AlMeu** [comment]
First be sure that you can connect to your unit and read it.
Without this you can't start do anything.

**Comment by kuduro88** [comment]
> > Please help to exit this virgin mode.
> 
> e-mail?

Hello, could you help me too? I'm in the same situation :(

my email is: heldermartins3@gmail.com

**Comment by misiek330** [comment]
Hey, you can help me because I tried to download the version from V2 to V3, unfortunately there is a black screen, I can't do anything email: misiek330@wp.pl

**Comment by AlMeu** [comment]
@misiek330 I answerd you by email.

**Comment by xalio1** [comment]
Hi all
I entered virgin mode and I have a black screen help me Renault ESPACE V 2015
I tried to do:
https://gitlab.com/py_ren/pyren/-/wikis/how%20to%20rlink2%20black%20screen
but it doesn't help
piotrekzajac1@wp.pl

**Comment by shrlnm** [comment]
@xalio1 
https://www.drive2.ru/b/643019689764615344/

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/issues/836 and one other....

**Comment by octavius3000** [comment]
Hi, following the Russian tutorial I managed to get an R-Link 2 working again that was blocked by pressing the Virgin Mode button, however there are functions that have not been restored.
The radio is connected to R-Link 2 of 8 slots, installed in a Koleos 2018. The software version is 3.3.16.986.
The existing problems are the following:
1. Rear view camera: The rear view camera is not active until the vehicle is in motion. However, if I go to radio settings in the Parking Assistance, the rear view camera transmits images. With ddt4all I managed to return the image to its normal orientation because the le appeared inverted.
2. Air conditioning: The air conditioning buttons are not visible, so the air conditioning flow cannot be adjusted. I tried to activate it from ddt4all, in the ECU CONFIGURATION / ADAS PRESENCE / ACC screen. I was in the present. I changed it to Not Present and then back to Present but with no positive results.
3. Station capture devices: Do not function, do not detect obstacles.
4. Navigation: It does not have the map I updated. I am not too worried about this as it will probably be corrected with the installation of the patch and the new map.
Thank you in advance for your valuable help.
Sorry for my english.
Thank you.
